modules/041-linstor/images/drbd-driver-loader/patches/README.md

```diff
@@ -0,0 +1,6 @@
+## Patches
+
+### Add change context directive for built modules
+
+Without it SELinux not allow to load kernel modules
+There is no PR in drbd repo, but it will be added in next release
```

Comment:
Please add link to upstream patch

https://lists.linbit.com/pipermail/drbd-dev/2023-May/007050.html